cmd/filestransfer/filestransfer.go

```diff
@@ -133,7 +134,7 @@ func main() {
 	fmt.Printf("File successfully downloaded from internal stage area to %v\n", tmpDir)
 
 	//Reading from downloaded file
-	file, err := os.Open(fmt.Sprintf("%v/data_to_upload.csv.gz", tmpDir))
+	file, err := os.Open(fmt.Sprintf("%v/%v.gz", tmpDir, filepath.Base(tmpFilePath)))
```

Comment:
would it make sense to do some `filepath.Join` instead of formatting with `/` since we already import the filepath module or it's irrelevant as `os.Open` is already platform agnostic?

Reply:
Good idea, fixed.